### ISSUE TITLE
Switch to `event-kit` instead of `emissary`

### ISSUE THREAD

**Issue by arthurvr** [Glavin001/atom-beautify#428]
From https://github.com/atom/emissary:

> Achtung! This library is currently used in Atom and various Atom dependencies, but our long-term plan is to transition away from it in favor of the simpler [event-kit](https://github.com/atom/event-kit) library. Don't depend on supporting this library forever.

**Comment by Glavin001** [comment]
+1 Thanks for letting me know!

**Comment by m1ga** [comment]
with the latest atom git version 1.2.0-dev I get this now:

~~~
Cannot find module '/usr/local/share/atom/resources/app.asar/node_modules/emissary'
Hide Stack Trace
Error: Cannot find module '/usr/local/share/atom/resources/app.asar/node_modules/emissary'
    at Module._resolveFilename (module.js:334:15)
    at Function.Module._resolveFilename (/usr/local/share/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:284:25)
    at Module.require (module.js:363:17)
    at require (module.js:382:17)
    at Object.<anonymous> (/home/miga/.atom/packages/atom-beautify/src/beautify.coffee:447:16)
    at Object.<anonymous> (/home/miga/.atom/packages/atom-beautify/src/beautify.coffee:2:1)
    at Module._compile (module.js:428:26)
    at Object.defineProperty.value [as .coffee] (/usr/local/share/atom/resources/app.asar/src/compile-cache.js:169:21)
    at Module.load (module.js:353:32)
    at Function.Module._load (module.js:308:12)
    at Module.require (module.js:363:17)
    at require (module.js:382:17)
    at Package.module.exports.Package.requireMainModule (/usr/local/share/atom/resources/app.asar/src/package.js:725:34)
    at Package.module.exports.Package.activateConfig (/usr/local/share/atom/resources/app.asar/src/package.js:250:14)
    at Package.module.exports.Package.activateNow (/usr/local/share/atom/resources/app.asar/src/package.js:228:14)
~~~

**Comment by Glavin001** [comment]
I would love to receive a Pull Request fixing this! Thanks!

**Comment by Glavin001** [comment]
I am running Atom v1.0.19 and things are still working.

Is anyone interested in submitting a Pull Request to fix this for 1.2.0-dev users?

**Comment by Glavin001** [comment]
Published fix to v0.28.15

Thank you everyone for your patience.
